src/test/java/org/xbill/DNS/NameTest.java

```diff
@@ -1249,6 +1249,14 @@ void toString_special_char() throws WireParseException {
     assertEquals(exp, n.toString());
   }
 
+  @Test
+  void toString_idn() throws TextParseException {
+    String in = "café.be";
```

Comment:
please use a subdomain of example.com/net/org to avoid any possible confusion with a real domain